src/components/Accordions/GeneralAccordion.jsx

```diff
@@ -0,0 +1,162 @@
+function GeneralAccordion() {
+  return (
+    <div className="accordion w-2/3 text-left mt-10" id="accordionExample">
+      <div className="flex flex-row justify-start align-center">
+        <div className="p-2">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            className="h-7 w-7"
+            viewBox="0 0 20 20"
+            fill="currentColor"
+          >
+            <path
+              fillRule="evenodd"
+              d="M18 10a8 8 0 11-16 0 8 8 0 0116 0zm-7-4a1 1 0 11-2 0 1 1 0 012 0zM9 9a1 1 0 000 2v3a1 1 0 001 1h1a1 1 0 100-2v-3a1 1 0 00-1-1H9z"
+              clipRule="evenodd"
+            />
+          </svg>
+        </div>
+        <p className="text-2xl font-semibold p-2">General</p>
+      </div>
+      <div className="accordion-item bg-[#FEEEEE] border border-gray-200">
```

Comment:
`Accordion Items` seems like they can be **one component** and used in many places.

Reply:
Actually I thought that idea but for first phase I separated them. In next time I'll use like that and thanks to props, I can give items text in props.

Reply:
> Actually I thought that idea but for first phase I separated them. In next time I'll use like that and thanks to props, I can give items text in props.

Let's keep this in mind and even create an issue for it to refactor our code a bit in the later stages @bayogluteoman. Thanks @haticecs for the great feedback. The only thing missing for this PR is tests.